openwisp_controller/config/admin.py

```diff
@@ -89,9 +93,34 @@ class Meta(BaseForm.Meta):
         model = Template
 
 
+def clone_selected_templates(modeladmin, request, queryset):
+    if request.user.is_superuser:
```

Comment:
this is not good enough, we must check the following:

- if there is more than one organization in the system AND
- if the user is superuser or if the user is member of more than one organization